static/docs/command-reference/checkout.md

```diff
@@ -16,60 +16,53 @@ positional arguments:
-which instance of each data file or directory should be used, with the checksums
-saved in the `outs` field. The `dvc checkout` command updates the workspace data
-to match with the <abbr>cached</abbr> files that correspond to those checksums.
-
-Using an SCM like Git, the DVC-files are kept under version control. At a given
-branch or tag of the SCM repository, the DVC-files will contain checksums for
-the corresponding data files kept in the cache. After an SCM command like
-`git checkout` is run, the DVC-files will change to the state at the specified
-branch or commit or tag. Afterwards, the `dvc checkout` command is required in
-order to synchronize the data files with the currently checked out DVC-files.
-
-This command must be executed after `git checkout` since Git doesn't track files
-that are under DVC control. For convenience a Git hook is available, simply by
-running `dvc install`, that will automate running `dvc checkout` after
-`git checkout`. See `dvc install` for more information.
-
-The execution of `dvc checkout` does:
-
-- Scan the `outs` entries in DVC-files to compare with the currently checked out
-  data files. The scanned DVC-files is limited by the listed `targets` (if any)
-  on the command line. And if the `--with-deps` option is specified, it scans
-  backward from the given `targets` in the corresponding
-  [pipeline](/doc/command-reference/pipeline).
-
-- For any data files where the checksum doesn't match their DVC-file entry, the
-  data file is restored from the cache. The link strategy used (`reflink`,
-  `hardlink`, `symlink`, or `copy`) depends on the OS and the configured value
-  for `cache.type` – See `dvc config cache`.
-
-Note that this command by default tries NOT to copy files between the cache and
-the workspace, using reflinks instead when supported by the file system. (Refer
-to
+The `dvc checkout` command synchronizes the workspace data to match with the
+current [DVC-files](/doc/user-guide/dvc-file-format) in the
+<abbr>project</abbr>. DVC knows which data files (a.k.a. <abbr>outputs</abbr>)
+to use because their checksums are saved in the `outs` fields inside the
+DVC-files.
+
+This is useful when the project is a <abbr>DVC repository</abbr>, since
+DVC-files versioned in different
+[revisions](https://git-scm.com/book/en/v2/Git-Internals-Git-References) will
+specify different outputs. When switching to those versions (with Git commands
+such as `git checkout`), the current DVC-files will no longer match with all of
+the data in the <abbr>workspace</abbr>, and so `dvc checkout` will be needed at
+that point.
+
+💡 For convenience, a Git hook is available to automate running `dvc checkout`
+after `git checkout`. Use `dvc install` to install it.
+
+The execution of `dvc checkout` does the following:
+
+- Scans the `outs` field values in DVC-files to compare with the outputs
+  currently in the <abbr>workspace</abbr>. Scanning is limited to the given
```

Comment:
it's not clear this way if targets limit outs or files? Let's be explicit and keep it simple.

Reply:
OK I see you did mean to remove that mention of `outs` field, at least. OK, simplified now.

Reply:
I guess this implies no need to mention checksums in the 2nd bullet either. Simplified that now too.

Wow, this brings us full circle back to my very first review that started this whole takeover! -> https://github.com/iterative/dvc.org/pull/864#discussion_r360175115

Reply:
p.s. updated --relink option desc. accordingly.